### PR TITLE
Change postinstall to use the installed version of bower

### PR DESCRIPTION
**Pull request by cwoloszynski**
The current repository's postinstall code did not work for me.  I found a comment in your issue list that suggested this fix.  I have implemented it and it worked for me.  

Here is a pull request to bring it over to your repository

### CODE REVIEW

**Comment by cwoloszynski** [comment]
Ok, so I am new to pull requests.  I prepped a second change (fixing the jsonwebtoken version) and was going to send a separate pull request but somehow this pull request has managed to include both of them.  I hope to learn how to do this better in the future.  In the meantime, I believe that this change brings the system into alignment with the current version of this package.

File: frontend/gulpfile.js [hunk 1]
```diff
@@ -345,8 +345,8 @@ function dist(ext, name, opt) {
     .pipe(opt.ngAnnotate ? g.rename : noop, name + '.annotated.' + ext)
     .pipe(opt.ngAnnotate ? gulp.dest : noop, './dist')
     .pipe(ext === 'js' ? g.uglify : g.minifyCss)
-    .pipe(g.rename, name + '.min.' + ext)
-    .pipe(gulp.dest, './dist')()
+    .pipe(g.rename, name + '.min.' + ext)()
+    //.pipe(gulp.dest, './dist')()
```
Comment (Tenkir) [inline frontend/gulpfile.js]: Why is there commented code?

**Comment by Tenkir** [comment]
Uncomment code then :cake:

File: frontend/gulpfile.js (hunk 1 above)
Comment (tarlepp) [inline frontend/gulpfile.js]: Hmm, does this work? What I see is that minified files aren't moved to ```dist``` folder anymore.

**Comment by tarlepp** [comment]
Any progress with this?

**Comment by tarlepp** [comment]
Any progress with this?

**Comment by tarlepp** [comment]
Any progress with this?

**Comment by tarlepp** [comment]
Any progress with this? Going to close this one if replies won't come.

**Comment by tarlepp** [comment]
no activity after questions, so closing this one.

**Pull request closed without merging**